dist/Makefile

```diff
@@ -0,0 +1,11 @@
+VERSION=1.8.0
+
+check:
+	rpmlint -i dist/c-jwt.spec
+
+rpm-prep:
+	mkdir -p ${HOME}/rpmbuild/SOURCES/
```

Comment:
I suggest using an rpm macro to find the rpmbuild root. It's not always set in ~/rpmbuild.